frontend/src/components/db/ParallelsChip.tsx

```diff
@@ -12,23 +13,18 @@ import CappedMatchesChip from "./CappedMatchesChip";
 export default function ParallelsChip() {
   const { t } = useTranslation("settings");
 
-  const { fileName, defaultQueryParams, queryParams } = useDbQueryParams();
-
-  // ignore some params that shouldn't result in refetching this query
```

Comment:
this had a fairly important role! Note to self: re-test when the text view is working again :)

Reply:
I understand, but it wasn't removed willy-nilly, it's that the way the params have been updated should made this point at this place moot. The comment is about removing params, but if you see what the filter are now (src/components/hooks/groupedQueryParams.ts) those params are never there to be removed. 

Happy to add a new note that expresses the general point about not re-fetching, but with the current structure it just doesn't seem applicable, as we only use the params relevant for an endpoint.

Reply:
I'll test it just to make sure it's still good :) But sounds great, thanks for taking care of this! 🙇